spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -18,13 +18,17 @@
 
 import java.util.regex.Pattern;
 
+import org.springframework.cloud.dataflow.core.dsl.ArgumentNode;
+
 /**
  * Sanitizes potentially sensitive keys for a specific command line arg.
  * @author Glenn Renfro
  */
 public class ArgumentSanitizer {
```

Comment:
I can't find it ATM, but I think there is a way in boot to modify which keys are considered the ones to sanitize.... @trisberg recently was asking how get boot to give up it's secrets.....   Did you find anything along that line of thought?